### PR TITLE
added a possible fix for OneSync compatibility using spectate & teleport

### PR DESCRIPTION
**Pull request by coleminer0112**
I have added a few simple 'hotfix' style lines of lua to get our server up and running with OneSync Infinity. Feel free to merger / use these additions as you see fit!

### CODE REVIEW

File: admin_server.lua [hunk 1]
```diff
@@ -269,7 +270,9 @@ Citizen.CreateThread(function()
 	AddEventHandler('EasyAdmin:requestSpectate', function(playerId)
 		if DoesPlayerHavePermission(source,"easyadmin.spectate") then
 			PrintDebugMessage("Player "..getName(source,true).." Requested Spectate to "..getName(playerId,true))
-			TriggerClientEvent("EasyAdmin:requestSpectate", source, playerId)
+			SendWebhookMessage(moderationNotification,string.format("%s has spectated %s", getName(source), getName(playerId)))
```
Comment (Blumlaut) [inline admin_server.lua:273]: you may want to add a new string to the translations here, instead of manually making one.

File: admin_server.lua [hunk 2]
```diff
@@ -539,6 +542,19 @@ Citizen.CreateThread(function()
 			TriggerClientEvent("EasyAdmin:TeleportRequest", playerId, px,py,pz)
 		end
 	end)
+
+	RegisterServerEvent("EasyAdmin:TeleportAdminToPlayer")
+	AddEventHandler("EasyAdmin:TeleportAdminToPlayer", function(id)
+		if GetPlayerName(id) and DoesPlayerHavePermission(source, "easyadmin.teleport") then
+			local tgtPlyr = id
```
Comment (Blumlaut) [inline admin_server.lua:549]: Plyr?

File: admin_server.lua [hunk 3]
```diff
@@ -539,6 +542,19 @@ Citizen.CreateThread(function()
...
+			local tgtPed = GetPlayerPed(tgtPlyr)
+			local tgtCoords = GetEntityCoords(tgtPed)
+			SendWebhookMessage(moderationNotification,string.format("**%s has teleported to %s**", getName(source), getName(id)))
```
Comment (Blumlaut) [inline admin_server.lua:552]: same critique as above, making a new string for it would be better.

**Review by Blumlaut — COMMENTED** [review]
Generally this seems like a decent pull request, however, i would like some changes to stay in line with the coding style ( e.g. replacing Plyr with Player or such, and adding the manually added text to the translation files instead.

**Comment by coleminer0112** [comment]
New commit resolves above critiques

**Pull request merged**